### PR TITLE
Add Types in Python tutorial

### PR DESCRIPTION
**Pull request by po5i**

### CODE REVIEW

File: tutorials/python/python-type-driven-development/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+---
+title: How to start using types in Python with Mypy
+published: 2020-01-29
+tags: Python, tutorial, typing, types, tdd
+language: Python
```
Comment (jpvillaisaza) [inline tutorials/python/python-type-driven-development/tutorial.md]: Does this work the same if you change it to `python`? There's another tutorial about Python that uses the lowercase version, but I don't remember if that does something different.

File: tutorials/python/python-type-driven-development/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+In this article, we’ll be walking through the Python type system - how to add types to our code and Docstrings and how to perform static type checking using [mypy][mypy]. There are plenty of options for static type-checking in Python. However, in this tutorial we will follow [Guido van Rossum's suggestion][guido-mypy].
+
+### Mypy
+
+Mypy is a third-party Python library that provides optional static type checking. Unlike other non-dynamic programming languages like Java, where the static type-checking takes place at compilation time, Mypy CLI does the type-check to a file (or a set of files) on-demand. Apart from type-checking at development time, it's helpful to also include this check automatically in the Continuous Integration pipeline.
+
+To start using mypy, install it using any version of `pip` globally or in your virtual environment.
```
Comment (jpvillaisaza) [inline tutorials/python/python-type-driven-development/tutorial.md]:
```suggestion
To start using mypy, install it using any version of `pip` globally or in your virtual environment:
```

File: tutorials/python/python-type-driven-development/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
+---
+title: How to start using types in Python with Mypy
```
Comment (jpvillaisaza) [inline tutorials/python/python-type-driven-development/tutorial.md]: Which one do we use for the URL? The filename or the title? If it's the filename, can you update it to match the title or the URL that we want?

**Review by jpvillaisaza — APPROVED** [review]
Looks good!

File: tutorials/python/python-type-driven-development/tutorial.md (hunk 3 above)
Comment (po5i) [reply tutorials/python/python-type-driven-development/tutorial.md]: Good catch. Updated.

File: tutorials/python/python-type-driven-development/tutorial.md (hunk 1 above)
Comment (po5i) [reply tutorials/python/python-type-driven-development/tutorial.md]: Changed to `python` to keep consistency.

**Comment by jpvillaisaza** [comment]
@po5i 👍

Can you merge after CI?

**Comment by po5i** [comment]
@jpvillaisaza Sure. How can we schedule the publishing of this one?

**Comment by jpvillaisaza** [comment]
@po5i We have to manually push to the `production` branch the day that we want to publish it.

**Review by flandrade — APPROVED** [review]

**Pull request merged**